src/box/iproto.cc

```diff
@@ -257,23 +257,6 @@ unsigned iproto_readahead = 16320;
 /* The maximal number of iproto messages in fly. */
 static int iproto_msg_max = IPROTO_MSG_MAX_MIN;
 
```

Comment:
This looks great, thanks @unera for the suggestion! Shame I didn't think of it.

The little problem though which I see is that all netbox connections now enable graceful shutdown. If some of our tests leave running requests on a server and then kill it, each such test will run 3 seconds longer multiplied by number of times it does this. Maybe we could make test-run and luatest set a smaller timeout as a default for us. We could also not care though, I am fine with it.